platforms/android/build_sdk.py

```diff
@@ -303,7 +236,6 @@ def gather_results(self, engines):
     parser.add_argument('--sign_with', help="Certificate to sign the Manager apk")
```

Comment:
Could you change 
parser.add_argument('--config', default='ndk-10.config.py',
to?
parser.add_argument('--config', default='ndk-17.config.py',